main/networking/Runtime.ts

```diff
@@ -104,36 +104,41 @@ function readPackets(data: Buffer, previousLeftoverBytes?: Buffer): { leftoverBy
  * Create TCP packet header and prepend to
  * payload to send to Runtime.
  */
-function createPacket(payload: unknown, messageType: MsgType): Buffer {
+function createPacket(payload: unknown, messageType: MsgType): Buffer | null {
   let encodedPayload: Uint8Array;
 
-  switch (messageType) {
-    case MsgType.DEVICE_DATA:
-      encodedPayload = protos.DevData.encode(payload as protos.IDevData).finish();
-      break;
-    case MsgType.RUN_MODE:
-      encodedPayload = protos.RunMode.encode(payload as protos.IRunMode).finish();
-      break;
-    case MsgType.START_POS:
-      encodedPayload = protos.StartPos.encode(payload as protos.IStartPos).finish();
-      break;
-    case MsgType.TIME_STAMPS:
-      encodedPayload = protos.TimeStamps.encode(payload as protos.ITimeStamps).finish();
-      break;
-    case MsgType.INPUTS:
-      encodedPayload = protos.UserInputs.encode({ inputs: payload as protos.Input[] }).finish();
-      break;
-    default:
-      console.log('ERROR: trying to create TCP Packet with unknown message type');
-      encodedPayload = new Uint8Array();
-      break;
-  }
+  try {
```

Comment:
Basically wrap this part in try/catch. If we catch an error then we return null since we couldn't encode the payload.